packages/api/src/routes/auth.ts

```diff
@@ -42,8 +44,26 @@ router.post("/signup", (req, res, next) => {
   if (typeof email !== "string" || typeof password !== "string") {
```

Comment:
Can we add an `ElNotification` on the frontend when a user signs up that says to check their email? should be easy to add to the `onSuccess` in `AuthSignupForm`